ADDONS_SUPPORT.md

```diff
@@ -1,19 +1,19 @@
 ## Addon / Framework Support Table
 
-| |[React](app/react)|[React Native](app/react-native)|[Vue](app/vue)|[Angular](app/angular)| [Polymer](app/polymer)| [Mithril](app/mithril)| [HTML](app/html)| [Marko](app/marko)|
-| ----------- |:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|
-|[a11y](addons/a11y)              |+| |+|+|+|+|+|+|
-|[actions](addons/actions)        |+|+|+|+|+|+|+|+|
-|[backgrounds](addons/backgrounds)  |+| |+|+|+|+|+|+|
-|[centered](addons/centered)      |+| |+|+| |+|+| |
-|[events](addons/events)          |+| |+|+|+|+|+|+|
-|[graphql](addons/graphql)        |+| | | | | | | |
-|[info](addons/info)              |+| | | | | | | |
-|[jest](addons/jest)              |+| | |+| | |+| |
-|[knobs](addons/knobs)            |+|+|+|+|+|+|+|+|
-|[links](addons/links)            |+|+|+|+|+|+|+| |
-|[notes](addons/notes)            |+| |+|+|+|+|+| |
-|[options](addons/options)        |+|+|+|+|+|+|+| |
-|[storyshots](addons/storyshots)  |+|+|+|+| | |+| |
-|[storysource](addons/storysource)|+| |+|+|+|+|+|+|
-|[viewport](addons/viewport)      |+| |+|+|+|+|+|+|
+| |[React](app/react)|[React Native](app/react-native)|[Vue](app/vue)|[Angular](app/angular)| [Polymer](app/polymer)| [Mithril](app/mithril)| [HTML](app/html)| [Marko](app/marko)| [Svelte](app/svelte)|
+| ----------- |:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|
+|[a11y](addons/a11y)              |+| |+|+|+|+|+|+| |
+|[actions](addons/actions)        |+|+|+|+|+|+|+|+|+|
+|[backgrounds](addons/backgrounds)  |+| |+|+|+|+|+|+|+|
+|[centered](addons/centered)      |+| |+|+| |+|+| |+|
+|[events](addons/events)          |+| |+|+|+|+|+|+| |
+|[graphql](addons/graphql)        |+| | | | | | | | |
+|[info](addons/info)              |+| | | | | | | | |
+|[jest](addons/jest)              |+| | |+| | |+| | |
+|[knobs](addons/knobs)            |+|+|+|+|+|+|+|+|+|
+|[links](addons/links)            |+|+|+|+|+|+|+| | |
+|[notes](addons/notes)            |+| |+|+|+|+|+| | |
+|[options](addons/options)        |+|+|+|+|+|+|+| | |
+|[storyshots](addons/storyshots)  |+|+|+|+| | |+| | |
```

Comment:
please mark all the supported addons

Reply:
I'm working my way through them, marking them off and committing as I go. I think I've marked off everything I've tested so far.